pkg/plugin/conntrack/_cprog/conntrack.c

```diff
@@ -68,6 +84,18 @@ struct ct_entry {
      * before retina deployment and the SYN packet was not captured.
      */
     bool is_direction_unknown;
+        /*
+        bytes_*_count indicates the number of bytes sent and received in the forward and reply direction.
+        These will be reset to 0 every time an event is reported.
```

Comment:
What event does this refer to?

Reply:
I've updated the comment as this was unclear. The counters relates to conntrack map entries, each counter is incremented based on processed packets for a given conntrack entry.